about.md

```diff
@@ -351,9 +351,9 @@ This is an invitation-based role. Facilitators are offered an honourarium in rec
 
 ### Our facilitators
 
-Our facilitator are essential for the program:
+Our facilitators are essential for the program:
 
-- [{{ ols-3-facilitators | uniq | size }} facilitators](/ols-3#facilitators) for [**OLS-3**]({% link ols-3.md %}) ({{ ols-3-schedule.weeks['01'].start }} - {{ ols-3-end }})
```

Comment:
@Esyteck These lines of code dynamically generate info about the number of facilitators involved in each cohort. This should not be classified as a typo.

However, your approach seems to cover the events of 1 or more facilitators...which is great. 👍

Let's see what @bebatut thinks.

Reply:
The suggestion look good